.github/workflows/release.yaml

```diff
@@ -42,45 +45,23 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Send a Slack notification if a publish happens
-        if: steps.changesets.outputs.published == 'true'
-        id: slack_notification
-        uses: slackapi/slack-github-action@v1.27.0
-        with:
-          payload: |
-            {
-              "blocks": [
-                {
-                  "type": "context",
-                  "elements": [
-                    {
-                      "type": "image",
-                      "image_url": "https://raw.githubusercontent.com/cloudoperators/juno/main/assets/logos/JUNO_logo_slack.png",
-                      "alt_text": "cloudoperators/juno"
-                    },
-                    {
-                      "type": "mrkdwn",
-                      "text": "JUNO Packages Released Successfully! 🚀"
-                    }
-                  ]
-                },
-                {
-                  "type": "rich_text",
-                  "elements": [
-                    {
-                      "type": "rich_text_preformatted",
-                      "elements": [
-                        {
-                          "type": "text",
-                          "text": ${{ toJson(steps.changesets.outputs.publishedPackages) }}
-                        }
-                      ],
-                      "border": 0
-                    }
-                  ]
-                }
-              ]
-            }
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+  notify-on-success:
+    if: needs.release.outputs.published == 'true'
+    needs: [release] # List the jobs that should complete successfully
+    uses: cloudoperators/juno/.github/workflows/shared-slack-notification.yaml@artie-shared-slack
+    with:
+      title: "JUNO Packages Released Successfully! 🚀"
+      body: ${{ needs.release.outputs.publishedPackages }}
+      rich_body: true
+    secrets:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  notify-on-failure:
+    if: failure()
+    needs: [release]
+    uses: cloudoperators/juno/.github/workflows/shared-slack-notification.yaml@artie-shared-slack
+    with:
+      title: "🚨 JUNO Supernova Image Failed 🚨"
```

Comment:
Why only Supernova? Shouldn't be JUNO package?

Reply:
Thx! @hodanoori